pandas/tests/io/json/test_pandas.py

```diff
@@ -1769,3 +1769,18 @@ def test_to_json_multiindex_escape(self):
             "\"(Timestamp('2017-01-23 00:00:00'), 'bar')\":true}"
         )
         assert result == expected
+
+    def to_json_series_of_objects(self):
+        class _TestObject:
```

Comment:
can you add the issue reference as a comment on this tests and the one below.

Reply:
The comment is slightly further down the test so it's closer to the expected output, is that ok?